Tests/UnitTests/TestHelpers/SnapshotTesting+Extensions.swift

```diff
@@ -86,8 +89,9 @@ extension SwiftUI.View {
 }
 #endif
 
-private let timeout: DispatchTimeInterval = .seconds(1)
-private let pollInterval: DispatchTimeInterval = .milliseconds(100)
+private let timeout: DispatchTimeInterval = .seconds(5)
+private let pollInterval: DispatchTimeInterval = .milliseconds(300)
+private let initialDelay: DispatchTimeInterval = .milliseconds(100)
```

Comment:
This is annoying, but necessary to make sure `SwiftUI`'s `task`s run and the snapshot is correct.